packages/nuxt-module/plugins/i18n.js

```diff
@@ -1,13 +1,18 @@
+import { useSharedState } from "@shopware-pwa/composables";
 import Vue from "vue";
 import VueI18n from "vue-i18n";
 
 Vue.use(VueI18n);
 
 export default ({ app }, inject) => {
+  const { sharedRef } = useSharedState(app);
+  const currentDomainData = sharedRef("sw-current-domain");
   // Set i18n instance on app
   // This way we can use it in middleware and pages asyncData/fetch
   const i18n = new VueI18n({
-    locale: "en-GB",
+    locale:
```

Comment:
I won't miss it anymore in CR ! :)

Reply:
It wouldn't work, either way, the other plugin is changing this value dynamically :)